gqueries/output_elements/output_series/user_uploaded_profiles/hydrogen_export_curve.gql

```diff
@@ -3,7 +3,7 @@
       V(
         FILTER(
         ALL(),
-        "hydrogen && (hydrogen.profile == :hydrogen_export)"
+        "hydrogen && (hydrogen.group == :hydrogen_export)"
```

Comment:
This does not match the group on the `energy_export_hydrogen_baseload` node. However, in this case we should update the group on the node to `hydrogen_export`, nice catch!